.yarnrc.yml

```diff
@@ -3,5 +3,7 @@ nodeLinker: node-modules
 plugins:
     - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
       spec: '@yarnpkg/plugin-workspace-tools'
+    - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+      spec: '@yarnpkg/plugin-interactive-tools'
```

Comment:
Would be cool to show this off to the team at some point! Maybe worth mentioning in #demo.

Reply:
Done!